README.md

```diff
@@ -76,9 +76,11 @@ Before you proceed, please check your ruby version by:
 
 ```sh
 $ ruby -v
-jruby 9.1.10.0 (2.3.3) 2017-05-25 b09c48a Java HotSpot(TM) 64-Bit Server VM 25.131-b11 on 1.8.0_131-b11 +jit [darwin-x86_64]
+jruby 9.1.12.0 (2.3.3) 2017-05-25 b09c48a Java HotSpot(TM) 64-Bit Server VM 25.131-b11 on 1.8.0_131-b11 +jit [darwin-x86_64]
```

Comment:
maybe we can remove the command output and simply say:

"Before you proceed, please check your ruby version by running `ruby -v`. The printed version should be the same as in the `.ruby-version` file."